customer-account-extension/src/OrderActionMenuItem.liquid

```diff
@@ -0,0 +1,28 @@
+{%- if flavor contains "react" -%}
+  import { Button, reactExtension, useApi } from '@shopify/ui-extensions-react/customer-account';
+
+  export default reactExtension(
+    "customer-account.order.action.menu-item.render",
+    () => <OrderActionMenuItem />
+  );
+  
+  function OrderActionMenuItem() {
+    const {i18n} = useApi()
+    return <Button to='/'>{i18n.translate("menuItemButton")}</Button>;
```

Comment:
I think we wanted this order action menu to redirect to the full page extension

Reply:
> I think we wanted this order action menu to redirect to the full page extension

Yes, When setting `to='/'`, it will be redirected to the full page extension after clicking it. @kirakik

Reply:
That is... very unintuitive. Is there a clearer way to do this we could use in the default example here?

Reply:
we can replace it with `extension:/` to make it more clear that we want to go to the root path of the full page extension